src/controller.coffee

```diff
@@ -958,6 +961,28 @@ define ['droplet-helper',
       # the hit test opportunity for this event.
       state.consumedHitTest = true
 
+  hook 'mousedown', 4, (point, event, state) ->
+    if state.consumedHitTest then return
+    if not @trackerPointIsInMain(point) then return
+
+    mainPoint = @trackerPointToMain point
+    hitTestResult = @hitTest mainPoint, @tree
+
+    if hitTestResult?
+      hitTestBlock = @view.getViewNodeFor hitTestResult
+      lineNumber = @findLineNumberAtCoordinate(mainPoint.y)
+
+      if hitTestBlock.addButtonRect? and hitTestBlock.addButtonRect.contains mainPoint
+        #console.log "Plus on: ", lineNumber
+        line = @mode.handleButton @getValue().split('\n')[lineNumber], 'add-button'
```

Comment:
I felt the same and suggested it few days back on the google group (https://groups.google.com/forum/#!msg/pencilcode-development/kcdiFv6GtdA/mTYMcqyX9EUJ), but since I didn't get an affirmation, I didn't work on it. Now that you have confirmed it, I went ahead and implemented it and you can see it in the latest commit.

Saksham